Courses.php

```diff
@@ -127,8 +127,19 @@ function extend_deadline(id) {
                     $type  = $_POST["type"];
                     $Deadline = $deadlinedate." ".$deadlinetime;
                     $date =  date("Y-m-d H:i");
+                    $targetfile="";
+                    $targetfile2="";
+                    $targetfile3="";
+                    $targetfile4="";
+                    if($_FILES['attachment1']['name']!=""){   $targetfile  = "/".$title."/".$_FILES['attachment1']['name']; }
+                    if($_FILES['attachment2']['name']!=""){   $targetfile2 = "/".$title."/".$_FILES['attachment2']['name']; }
+                    if($_FILES['attachment3']['name']!=""){   $targetfile3 = "/".$title."/".$_FILES['attachment3']['name']; }
+                    if($_FILES['attachment4']['name']!=""){   $targetfile4 = "/".$title."/".$_FILES['attachment4']['name']; }
```

Comment:
@Jeannick94 

Where will the attachment be stored on the server/computer?

-Hui

Reply:
@spm2020spring 

Happy new year Sir,

On the computer/server, the attachment will be stored in the "Lab_Report_Assignments" folder, and for more details, you can open the following attachments:
![LRR3](https://user-images.githubusercontent.com/73539808/103448492-3f610500-4cd5-11eb-80d8-2c2ad236fbef.JPG)
![LRR4](https://user-images.githubusercontent.com/73539808/103448577-61a75280-4cd6-11eb-8fd5-dff003f9ffbf.JPG)

-Jeannick U.